src/gg/codie/mineonline/discord/DiscordChatBridge.java

```diff
@@ -0,0 +1,92 @@
+    String Channel = "";
+    String Token = "";
+    String Vers = "";
+    JDA jda;
+
+    public DiscordChatBridge(String version, String discordChannel, String discordToken, String webhookUrl, MessageRecievedListener msgEvent) {
+
+        Token = discordToken;
+        Channel = discordChannel;
+        Webhook = webhookUrl;
+
+        if (version.startsWith("Classic")) // Jank color character switcher for old version, may fail on inf/indef
+            Vers = "&";
+        else
+            Vers = "§";
+
+        if (Webhook != null) {
+            try {
+                builder = new WebhookClientBuilder(Webhook);
+                client = builder.build();
+            } catch (Exception ex) {
+                System.out.println(ex);
+            }
+        }
+            try {
+
+                jda = JDABuilder.createDefault(Token)
+                        .build();
+                jda.awaitReady();
+                System.out.println("Discord Bridge Started");
+                jda.addEventListener(new ListenerAdapter() {
+                    @Override
+                    public void onMessageReceived(MessageReceivedEvent event) {
+                        if(event.getChannel().getId().equals(Channel) && !event.isWebhookMessage() && !event.getAuthor().getId().equals(event.getJDA().getSelfUser().getId())) { // stop listening to yourself
+                            message = ("say " + event.getAuthor().getName() + ": " + Vers + "f" + event.getMessage().getContentStripped());
+                            msgEvent.onMessageRecieved(message);
+                        }
+                    }
+                });
+
+            } catch (Exception ex) {
+                System.out.println(ex);
+            }
+    }
+
+    public void sendDiscordMessage(String username, String message){
+        if (Webhook != null && !username.equals("")){ // Webhook player messages to discord
+            webhookMessage = new WebhookMessageBuilder();
+            webhookMessage.setUsername(username);
+            webhookMessage.setAvatarUrl("https://minotar.net/cube/" + username + "/100.png");
+            webhookMessage.setContent(message);
+            client.send(webhookMessage.build());
+        } else if (Token != null && !username.equals("")) { // Non-webhook player messages to discord
+            try {
+                long chan = Long.parseLong(Channel);
+                channel = jda.getTextChannelById(chan);
+                channel.sendMessage("**" + username + "**: " + message).queue();
+            } catch (Exception ex) {
+                System.out.println(ex);
+            }
+        } else if (username.equals("")){ // Non webhook system messages to discord
+            try {
+                long chan = Long.parseLong(Channel);
+                channel = jda.getTextChannelById(chan);
+                channel.sendMessage(message).queue();
+            } catch (Exception ex) {
+                System.out.println(ex);
```

Comment:
Please use ex.printStackTrace() instead. Though this is generally considered bad practice (for reasons I do not yet know), it'd be better to remain consistent for now so that when these calls are all swapped in future, we don't miss any.